MANIFEST.in

```diff
@@ -0,0 +1,8 @@
+include README.md LICENSE settings.yml
+recursive-include buildtest *
+recursive-include buildtest *.yml
+graft template
```

Comment:
there is no ``template`` directory in root of buildtest. maybe you meant to say ``graft buildtest``

Reply:
I think this was previously in the repository, removed, and I added back - I don't recall adding this graft line because I'm not familiar with the instruction. But I'm happy to remove it!